.github/workflows/deploy.yml

```diff
@@ -39,10 +40,20 @@ jobs:
       if: github.ref == 'refs/heads/beta'
       run: echo "BETA=true" >> $GITHUB_ENV
 
-    - name: Use scripts and templates from master branch
+    - name: Use deploy script from master branch
       run: |
         git fetch --no-tags --prune --depth=1 origin master
-        git checkout origin/master -- .github/deploy.sh util/gh-pages/ util/*.py
+        git checkout origin/master -- .github/deploy.sh util/versions.py util/gh-pages/versions.html
```

Comment:
This checkout is missing the `util/gh-pages/index.html` for the lint list itself. The `index.html` is referenced in `deply.sh:10`

Reply:
This is intentional. If we'd check that out, we'd get weird incompatibilities between the output of the metadata collector output and the index.html file. This file still exists in the beta branch and on tags.

Reply:
Okay, so this is basically intended for the next version until the new metadata collector is also used for beta and tags, right?

Reply:
Nope, we have to keep it like this, since we can't check out a newer version of the metadata collector anymore. Before we checked out the documentation generation script ``export.py`` **and** the ``index.html`` so it was always in a consistent state.

Reply:
Is the deploy script working in a clone of the repo?

Reply:
Yes

Reply:
That makes sense now. Thank you for clarifying. 

I guess @flip1995 r+ as you wanted to merge this into a repo branch. ^^